### DISCUSSION TITLE
Welcome to kube-prometheus Discussions!

### DISCUSSION THREAD

**Discussion by metalmatze** [prometheus-operator/kube-prometheus#593] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bwplotka** [comment, marked as answer]
![](https://media3.giphy.com/media/3o7TKSjRrfIPjeiVyM/giphy.gif)
